packages/docs/docs/dev/hooks-reference.mdx

```diff
@@ -5,6 +5,36 @@ category: For Developers
 
 # React Hooks Reference
 
+## `useContractRead`
+
+Reads and calls a method from a Fuel contract, returns `<InvokeFunctions>`.
+
+
+#### Reading with a Contract instance
```

Comment:
h5 looks better imo

```suggestion
##### Reading with a Contract instance
```

Reply:
I'll use h5 in my docs too (for those subsections)

<img width="370" alt="Screenshot 2024-06-03 at 13 28 07" src="https://github.com/FuelLabs/fuels-wallet/assets/7074983/8253afa8-75f9-4898-a8b1-0d75ba47404f">

Reply:
Good one, fixed